README.md

```diff
@@ -1,11 +1,22 @@
-The main website for hs3.pl.
+# The main website for hs3.pl
 
-In order to run the website locally:
+## In order to run the website locally
 
-- Install hugo: https://gohugo.io/installation/
+- Install hugo extended edition: https://gohugo.io/installation/
 - (Fork &) clone repo
 - Do a `git submodule update --init`
 - `hugo serve`
 - Go to http:/localhost:1313/
 
 Also read the [I18n notes](i18n.md) and the [calendar/event notes](calendar.md).
+
+## To contribute
+
+There are 2 main ways of contributing to the website.
+
+1. Create a PR with proposed changes from your fork of the repository.
+2. If you want to make a contribution in the ["projekty"](https://hs3.pl/projekty/) or ["zasoby"](https://hs3.pl/zasoby/) subpage, you can do so via [Decap CMS GUI](https://decapcms.org/). Go to https://hs3.pl/admin, create a personal account and add your contribution.
+
+Both ways of contribution require a review from the website's maintainers before the changes can be integrated.
+
+You can acccess Decap CMS via localhost at localhost:1313/admin. Be mindful that no matter where you access Decap CMS - whether running locally, or in your published site — it will always fetch and commit files in the hosted repository. You can read more about it [here](https://decapcms.org/docs/configuration-options/).
```

Comment:
The later warning looks good, but not sure there is any reason to run Decap on localhost, right? Maybe not necessary to add here.

Reply:
I also think there is no reason to run Decap locally - i removed the first part. I just wanted to make sure that in this case a user knows that they will create a PR on remote. 🙂

In terms of specifying the repository, if I understand the [documentation](https://decapcms.org/docs/configure-decap-cms/#backend) correctly, most of it is done via Netlify. So we should have everything needed in the config file, we just need to specify the following:
```
backend:
  name: git-gateway
  branch: main
```
But someone with access to Netify of hs3 website will have to add the necessary settings.

Reply:
So to me those two statements contradict each other.
If it's some magic in Netlify, I don't see how it could ever work on localhost (when I run the site on localhost, it doesn't ever touch Netlify).

Maybe best to just merge it and see what happens, however I have a gut feeling that something is missing

Reply:
Okay, yes, I'd be grateful if we could try it out. I don't really see what I could add to the configuration file right now. 

I succeded in creating a PR to my fork before, so I hope it will work here as well. I linked my fork in the `Continous Deployment` on Netlify. This is one of the PRs: https://github.com/MartaSien/hs3.pl/pull/13